mypy/checkexpr.py

```diff
@@ -2088,7 +2091,8 @@ def analyze_external_member_access(self, member: str, base_type: Type,
     def is_literal_context(self) -> bool:
         return is_literal_type_like(self.type_context[-1])
 
-    def infer_literal_expr_type(self, value: LiteralValue, fallback_name: str) -> Type:
+    def infer_literal_expr_type(self, value: LiteralValue, fallback_name: str,
+                                fallback_typ: Optional[Instance] = None) -> Type:
```

Comment:
This signature is now a bit awkward. I think that it would be better to always only take an `Instance`, and some callers can be updated to use `self.named_type`.